sdcm/mgmt.py

```diff
@@ -350,6 +350,20 @@ def wait_for_status(self, list_status, check_task_progress=True, timeout=3600, s
                                           timeout=timeout)
         return is_status_reached
 
+    def wait_for_percentage(self, minimum_percentage, timeout=3600, step=10):
+        text = f"Waiting until task: {self.id} reaches at least {minimum_percentage}% progress"
+        is_percentage_reached = wait.wait_for(func=self.has_progress_reached_percentage,
+                                              minimum_percentage=minimum_percentage,
+                                              step=step, text=text, timeout=timeout)
+        return is_percentage_reached
+
+    def has_progress_reached_percentage(self, minimum_percentage):
```

Comment:
when adding such utility, please add a docstring, explain what this utility is going todo, and **why** it's needed.